radicle-node/src/service.rs

```diff
@@ -156,9 +158,13 @@ pub enum Command {
     UntrackNode(NodeId, chan::Sender<bool>),
     /// Query the internal service state.
     QueryState(Arc<QueryState>, chan::Sender<Result<(), CommandError>>),
+    /// Worker has completed fetch job
+    FetchCompleted(Id, NetTransport<NoiseXk<G>, Message>),
+    /// Worker has failed fetch job
+    FetchFailed(Id, storage::Error, NetTransport<NoiseXk<G>, Message>),
```

Comment:
These should be methods on `Service`, as they aren't RPC calls but rather events that happened.